src/plugins/custom_integrations/public/components/fleet_integration/elasticsearch_js/elasticsearch_js_readme.tsx

```diff
@@ -124,42 +122,6 @@ export const ElasticsearchJsClientReadme = () => {
                 </EuiCodeBlock>
               </EuiPageSection>
 
-              <EuiPageSection>
```

Comment:
I have a question about this, do you intend to remove the button for the js client only or also for all the other clients?